src/test/resources/com/puppycrawl/tools/checkstyle/checks/design/hideutilityclassconstructor/InputHideUtilityClassConstructorIgnoreAnnotationBy.java

```diff
@@ -0,0 +1,46 @@
+/*
+HideUtilityClassConstructor
+ignoreAnnotatedBy = Skip, SkipWithParam, SkipWithAnnotationAsParam
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.design.hideutilityclassconstructor;
+
+@Skip
+public class InputHideUtilityClassConstructorIgnoreAnnotationBy {
+  public static void func() {}
+}
+
+@SkipWithParam(name = "tool1")
+class ToolClass1 {
+  public static void func() {}
+}
+
+@SkipWithAnnotationAsParam(skip = @Skip)
+class ToolClass2 {
+  public static void func() {}
+}
+
+@CommonAnnot
+@Skip
+class ToolClass3 {
+  public static void func() {}
+}
+
+@CommonAnnot // violation, should not have a public or default constructor
```

Comment:
`// violation, 'should not have a public or default constructor'`

Reply:
Without quotes it matches is not a violation message is just general reason that explains violation in author words